samples/sampler/stories/qa/MenuItem.module.less

```diff
@@ -0,0 +1,10 @@
+.menuItem {
+	.iconBefore {
```

Comment:
Is there a case where iconBefore is displayed in the sampler?
Isn't this also unnecessary?